app/src/pages/registrationPage/registrationPage.tsx

```diff
@@ -19,14 +18,15 @@ export const RegistrationPage = (props: IRegistrationPageProps): React.ReactElem
   const [retypedPassword, setRetypedPassword] = React.useState<string | null>(null);
   const [isLoading, setIsLoading] = React.useState<boolean>(false);
 
-  const onRegisterClicked = (): void => {
+  const onRegisterClicked = async (): Promise<void> => {
```

Comment:
this small change fixes the loading overlay on the registration page

Reply:
nice